schema/v1alpha1/system.proto

```
@@ -5,307 +5,1138 @@ syntax = "proto3";
 
-  // Physical location of the management controller
-  string physical_location = 7;
+// Log filter
+message LogFilter {
+  // Filter name
+  string name = 1;
 
-  // Status of the management controller
-  ManagementControllerStatus status = 8;
+  // Filter pattern
+  string pattern = 2;
```

Buf (1.55.1)

symbol "schema.v1alpha1.EnvironmentalSpecs" already defined at schema/v1alpha1/location.proto:233:9

(COMPILE)